integrations/clickup/main.py

```diff
@@ -0,0 +1,117 @@
+from enum import StrEnum
+from loguru import logger
+from clickup.client import ClickUpClient
+
+from port_ocean.context.ocean import ocean
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+
+class ObjectKind(StrEnum):
+    """Defines the different kinds of objects in the system."""
+    TEAM = "team"
+    PROJECT = "project"
+    ISSUE = "issue"
+
+
+async def setup_application() -> None:
+    """
+    Sets up the application by creating necessary webhooks.
+
+    If no app host is provided, it logs a warning and skips the webhook creation.
+    """
+    logic_settings = ocean.integration_config
+    app_host = logic_settings.get("app_host")
+
+    if not app_host:
+        logger.warning(
+            "No app host provided, skipping webhook creation. "
+            "Without setting up the webhook, the integration will not export live changes from ClickUp."
+        )
+        return
+    #webhook  setup goes here
```

Comment:
If you are not subscribing to webhook then there is no reason to keep this function